node/src/Pos/Web/Server.hs

```diff
@@ -167,7 +166,7 @@ getUtxo :: HasConfiguration => WebMode ssc [TxOut]
 getUtxo = map toaOut . toList <$> GS.getAllPotentiallyHugeUtxo
 
 getLocalTxsNum :: WebMode ssc Word
-getLocalTxsNum = fromIntegral . length <$> getLocalTxs
+getLocalTxsNum = fromIntegral . length <$> withTxpLocalData getLocalTxs
```

Comment:
Is this a top level thing, or should this also be an STM transaction (i.e. also move the `withTxpLocalData` to the caller).

Reply:
It is a top level thing.